AnnService/src/Core/BKT/BKTIndex.cpp

```diff
@@ -196,7 +196,7 @@ namespace SPTAG
                 SizeType nn_index = node[i]; \
                 if (nn_index < 0) break; \
                 if (p_space.CheckAndSet(nn_index)) continue; \
-                float distance2leaf = m_fComputeDistance(p_query.GetTarget(), (m_pSamples)[nn_index], GetFeatureDim()); \
+                float distance2leaf = m_fComputeDistance(p_query.GetQuantizedTarget(), (m_pSamples)[nn_index], GetFeatureDim()); \
```

Comment:
I think it is cleaner to describe it this way, because when we have no quantizer set, we can think of it as the 'identity quantizer', that quantizes every vector to itself. Also, it is easier to make default option (GetTarget) most compatible (reconstructed vector), and explicitly switch to GetQuantizedTarget when we only need distance calculation ability. 

If you have strong opinion on it, I am willing to change it though